cdap-ui/app/features/admin/admin.less

```diff
@@ -88,4 +88,10 @@ body.theme-cdap.state-admin {
       }
     }
   }
+
+  .modal-body {
```

Comment:
I am targeting the panel-title for the accordion (under modal). Without it, the text color is white. I am guessing this styling will get changed when we do the real styling for modals.